youtube_dlc/extractor/common.py

```diff
@@ -1868,6 +1870,8 @@ def _parse_m3u8_formats(self, m3u8_doc, m3u8_url, ext=None,
             if re.match(r'^https?://', u)
             else compat_urlparse.urljoin(m3u8_url, u))
 
+        ignore_discontinuity = self._downloader.params.get('hls_ignore_discontinuity')
```

Comment:
```suggestion
        ignore_discontinuity = self._downloader.params.get('hls_ignore_discontinuity', True)
```

Keep the old behavior by default. We don't want any scripts/URLs that used to work correctly to break